system76_acpi.c

```diff
@@ -690,6 +683,9 @@ static int system76_add(struct acpi_device *acpi_dev)
 	acpi_dev->driver_data = data;
 	data->acpi_dev = acpi_dev;
 
+	data->has_open_ec = acpi_has_method(acpi_device_handle(data->acpi_dev),
```

Comment:
Not sure this would actually work

Reply:
Do we have a way to know if we're running open EC firmware from this driver?

Reply:
It seems like you found a way, is that right?

Reply:
If someone tells me it works.